api/types/resource.go

```diff
@@ -139,6 +139,18 @@ type EnrichedResource struct {
 	RequiresRequest bool
 }
 
+// EnrichedResources is a list of Enriched resources.
```

Comment:
But what is an enriched resource?

Reply:
i have changed it to `EnrichedResources is a wrapper of []*EnrichedResource.`. any other recommendations?

Reply:
[EnrichedResource](https://github.com/gravitational/teleport/blob/master/api/types/resource.go#L131-L140) is a resource that has had `Logins` and `RequiresRequest` added to it.